src/js/mainSiteURLs.json

```diff
@@ -0,0 +1,39 @@
+[
```

Comment:
Making this JSON assures we only load it when the service worker is installed. Before, we were loading it every time the service worker became active.